index.html

```diff
@@ -138,29 +138,48 @@ <h2>
         <p>All recipients will be awarded a ticket to attend the conference and conference affiliated evening programming as well as being set up with a mentor to answer questions and help guide recipients through the event. This year we are proud to be able to offer a limited amount of travel assistance. Find all the details and fine print in the <a href="https://goo.gl/forms/HvqoFx6JKFGnCY5i2">application</a>.</p>
         <p>Individuals who would like to sponsor a scholarship can <a href="https://ti.to/empirejs/empire-2018">make a diversity and opportunity donation here</a>. Organizations can find more information on how to <a href="https://2018.empirejs.org/EmpireJS-2018-prospectus.pdf">become a sponsor here</a>.</p>
       </section>
-      
+
       <section id="sponsors" class="text-section">
         <h2>
           <img class="scribble--sponsors" alt="" src="scribbles/blue-scribble.svg">
           <span>Sponsors</span>
         </h2>
         <h3>Platinum</h3>
           <ul class="sponsors-list">
-            <li class="sponsor-box"><img class="sponsor-logo" src="sponsor-logos/ibm.jpg"></li>
+            <li class="sponsor-box">
+              <a href="https://developer.ibm.com/"><img class="sponsor-logo" src="sponsor-logos/ibm.jpg"></a>
+            </li>
           </ul>
 
+        <h3>Gold</h3>
+        <ul class="sponsors-list">
+          <li class="sponsor-box">
+            <a href="https://vrworldnyc.com/"><img class="sponsor-logo add-padding" src="sponsor-logos/vrworld.png"></a>
+          </li>
+        </ul>
+
         <h3>Silver</h3>
         <ul class="sponsors-list">
-          <li class="sponsor-box"><img class="sponsor-logo add-padding" src="sponsor-logos/slice.png">
-          </li><li class="sponsor-box"><img class="sponsor-logo add-padding" src="sponsor-logos/microsoft.png">
-          </li><li class="sponsor-box"><img class="sponsor-logo add-padding" src="sponsor-logos/mongo.svg">
-          </li><li class="sponsor-box"><img class="sponsor-logo add-padding" src="sponsor-logos/glossier.jpg"></li>
+          <li class="sponsor-box">
+            <a href="https://slicelife.com/"><img class="sponsor-logo add-padding" src="sponsor-logos/slice.png"></a>
+          </li>
+          <li class="sponsor-box">
+            <a href="https://www.microsoft.com/"><img class="sponsor-logo add-padding" src="sponsor-logos/microsoft.png"></a>
```

Comment:
According to the list of links in the Playbook, Microsoft's is https://developer.microsoft.com/en-us/advocates/index.html